requirements.txt

```diff
@@ -1,20 +1,20 @@
 attrs==19.3.0
 certifi==2023.7.22
-cffi==1.14.0
+cffi>=1.15.1
 chardet==3.0.4
-click==7.1.2
+click>=8.0
 cryptography==42.0.4
 Flask==2.2.5
 idna==2.9
-importlib-metadata==1.6.0
-itsdangerous==1.1.0
+importlib-metadata>=3.6.0
+itsdangerous>=2.0
 Jinja2==3.1.3
 jsonschema==3.2.0
-MarkupSafe==1.1.1
+MarkupSafe>=2.1.1
 pycparser==2.20
 pyOpenSSL==19.1.0
 pyrsistent==0.16.0
-PyYAML==5.4
+PyYAML>=5.3.1
```

Comment:
did you mean to lower the minimum version for `PyYAML`?

Reply:
Good catch. Let me test that out and see if I can use a more recent version.

Reply:
It looks like there is an issue with the `5.4` versions and I had to either bump this to `6.0` or `5.3` and `6.0` works in my testing. So, I've one with that.

The new commit takes a slightly different approach and constrains these dependencies to their major version via `~=` which seems like a safer approach than `>=`.